rele/management/commands/document.py

```diff
@@ -0,0 +1,30 @@
+from django.apps import apps
+from django.core.management import BaseCommand
+from django.utils.module_loading import module_has_submodule
+from tabulate import tabulate
+
+from rele.config import load_subscriptions_from_paths
+
+
+def discover_subs_modules():
```

Comment:
I think we have a very similar function in the file runrele.py `_discover_subs_modules`. What do you think about to move this function somewhere and use the same function in both cases?

Reply:
I agree with you @edgarlatorre . I was aware this function was duplicated but I didn't know how to remove that duplication. What if we move this function to `rele.config` module? Do you have any suggestions? 🤔

Reply:
We could create a `ReleBaseCommand` and use it in both commands. That would be my suggestion.

Reply:
My first thought would move the function to `rele.config`.  I wouldn't create a base command for that as `discover subscription` is not responsibility of a command.

Reply:
The issue with moving it to `rele.config` is that it has django django dependencies. And we don't want to pollute what will eventually be rele standalone with more django dependencies.  

The other option I could see is moving it to a `rele.management.commands.discover` module.

Reply:
What about to have something like a loader.py module? I believe it'll be necessary to use this function for a standalone version.

Reply:
This could be moved to `rele.contrib.django`

Reply:
I like what @andrewgy8 says about keeping Django dependencies as isolated as possible and agree with @edgarlatorre in that it makes little sense to create a `BaseCommand` subclass only for adding this function. 

I'll try to move the function to `rele.management.commands.discover` module so that the relative import statement is short enough. Let's see how it looks 😄